### ISSUE TITLE
Updates tab shows duplicates of already installed versions

### ISSUE THREAD

**Issue by jhnnslmk** [sebescudie/GammaLauncher#43]
So I already have the 2022.5.0-0558 preview installed:
<img width="114" alt="image" src="https://user-images.githubusercontent.com/1488726/217217130-a6a62452-6eef-4c4f-ae1d-8f2f6b0831e0.png">

So in a perfect world I'd expect to see its entry greyed out:
<img width="165" alt="image" src="https://user-images.githubusercontent.com/1488726/217217287-bdf96063-ca73-4bf6-913e-415df1da8245.png">

And even better, some kind of indicator on top (just like you have it in place for the launcher itself already) if there's more recent vvvv versions available :)

**Comment by sebescudie** [comment]
Agreed, the _Updates_ tab, is rather stupid : it does not care about what you have installed or not, it'll just show the last five builds per build config on TeamCity. 

Rather than greying out versions you already have, I'd rather have the ones you don't have highlighted!
